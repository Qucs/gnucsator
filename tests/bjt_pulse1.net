
BJT:Q2N3904_1 in c out c Type="npn" Is="1.4e-14" Nf="1" Nr="1" Ikf="0.025" Ikr="0" Vaf="100" Var="0" Ise="3e-13" Ne="1.5" Isc="0" Nc="2" Bf="300" Br="7.5" Rbm="0" Irb="0" Rc="2.4" Re="0" Rb="0" Cje="4.5e-12" Vje="0.75" Mje="0.33" Cjc="3.5e-12" Vjc="0.75" Mjc="0.33" Xcjc="1" Cjs="0" Vjs="0.75" Mjs="0" Fc="0.5" Tf="4e-10" Xtf="0" Vtf="0" Itf="0" Tr="2.1e-08" Temp="26.85" Kf="9e-16" Af="1" Ffe="1" Kb="0" Ab="1" Fb="1" Ptf="0" Xtb="1.5" Xti="3" Eg="1.11" Tnom="26.85" Area="1"
Vdc:V1 c gnd U="5 V"
R:R3 gnd out R="100 Ohm" Temp="26.85" Tc1="0.0" Tc2="0.0" Tnom="26.85"
Relais:XX gnd out gnd gnd Vt="1.2" Vh="1"
Vpulse:V2 in gnd U1="0 V" U2="3 V" T1="0" T2="1 ms" Tr=".1 ms" Tf=".1 ms"
.TR:TR1 Type="lin" Start="0" Stop="1.5 ms" Points="1000" IntegrationMethod="Trapezoidal" Order="2" InitialStep="1 ns" MinStep="1e-16" MaxIter="150" reltol="0.001" abstol="1 pA" vntol="1 uV" Temp="26.85" LTEreltol="1e-3" LTEabstol="1e-6" LTEfactor="1" Solver="CroutLU" relaxTSR="no" initialDC="yes" MaxStep="0"
