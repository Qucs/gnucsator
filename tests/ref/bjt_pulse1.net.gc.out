<Qucs Dataset>
default plugins: ignored
;: already installed, replacing
stashing as ;:0
*: already installed, replacing
stashing as *:0
': already installed, replacing
stashing as ':0
": already installed, replacing
stashing as ":0
store: already installed, replacing
stashing as store:0
alarm: already installed, replacing
stashing as alarm:0
iplot: already installed, replacing
stashing as iplot:0
plot: already installed, replacing
stashing as plot:0
iprint: already installed, replacing
stashing as iprint:0
print: already installed, replacing
stashing as print:0
probe: already installed, replacing
stashing as probe:0
d: already installed, replacing
stashing as d:0
diode: already installed, replacing
stashing as diode:0
npn: already installed, replacing
stashing as npn:0
pnp: already installed, replacing
stashing as pnp:0
npn1: already installed, replacing
stashing as npn1:0
pnp1: already installed, replacing
stashing as pnp1:0
Q: already installed, replacing
stashing as Q:0
bjt: already installed, replacing
stashing as bjt:0
L: already installed, replacing
stashing as L:0
R: already installed, replacing
stashing as R:1
Gnucap   System status
iterations: op=0, dc=0, tran=178, fourier=0, total=217
transient timesteps: accepted=20, rejected=19, total=39
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=3, subckt=0, model=1, total=4
dctran density=100.0%, ac density=100.0%

<indep time 20>
 0.
 3.E-12
 70.46723E-6
 94.65169E-6
 94.66752E-6
 94.66752E-6
 100.E-6
 177.7778E-6
 219.0972E-6
 260.4167E-6
 329.2824E-6
 0.0011
 0.001175122
 0.001175122
 0.0012
 0.001200142
 0.001200284
 0.00120039
 0.001200495
 0.0015
</indep>
<dep c.Vt time>
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
 5.
</dep>
<dep in.Vt time>
 0.
 89.99979E-9
 2.114017
 2.839551
 2.840026
 2.840026
 3.
 3.
 3.
 3.
 3.
 3.
 0.7463447
 0.7463446
 956.E-15
-948.E-15
 941.E-15
-931.E-15
 921.E-15
-921.E-15
</dep>
<dep out.Vt time>
 1.096044E-9
 90.4993E-9
 1.38916
 2.099533
 2.1
 2.1
 2.2572
 2.2572
 2.2572
 2.2572
 2.2572
 2.2572
 0.09999996
 0.0999999
-5.933903E-6
 5.861375E-6
-5.785342E-6
 5.689701E-6
-5.591245E-6
 5.593454E-6
</dep>
